emData/download.sh

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env bash
 
+# Standard configuration
+memprints_url="https://www.dropbox.com/s/nvuthntfb68lgxz/MemPrints_Standard_210514.tgz?dl=0"
+luts_url="https://www.dropbox.com/s/54jbuihchjagzkh/LUTs_Standard_210514.tgz?dl=0"
+# Combined modules
+memprints_url_cm="https://www.dropbox.com/s/92uwmigihji4b5e/MemPrints_Combined_210514.tgz?dl=0"
+luts_url_cm="https://www.dropbox.com/s/8ff6dfysgnrxycx/LUTs_Combined_210514.tgz?dl=0"
+
+
 #### fw_synch_210503 ####
 # Standard configuration
-memprints_url="https://cernbox.cern.ch/index.php/s/CipX7CfTXIj1lcK/download"
-luts_url="https://cernbox.cern.ch/index.php/s/UDSvClVZksBr1Pq/download"
+#memprints_url="https://cernbox.cern.ch/index.php/s/CipX7CfTXIj1lcK/download"
```

Comment:
any reason to not just delete these commented out lines?

Reply:
I'm OK with removing the commented out lines. Alex(?) had left the versions of the files that corresponds to the two earlier tags commented out so I followed this pattern. But I can remove these commented out version if that is what we prefer.

Reply:
I kind of prefer keeping the URLs of past tags around. I realize the top of this file may wind up getting pretty cluttered, but I like preserving easy access to these historical test-vectors, just in case.

Reply:
the old URLs are always available in the git history, of course...

Reply:
btw why are some in cernbox and some in dropbox?

Reply:
I put my temporary (i.e. not corresponding to a tag) tarballs on dropbox.

Reply:
I don't feel strongly about keeping the commented-out URLs here. So if there's a desire to remove them, I'm OK with it.

Reply:
@aehart I don't feel strongly but don't really see a reason to keep them. but you're the one doing the actual work so I defer to you.

Reply:
Let's keep the commented-out URLs for now. We can always reconsider in a future PR.